test/irb/test_init.rb

```diff
@@ -50,6 +50,27 @@ def test_rc_file
       end
     end
 
+    def test_rc_file_loading_order
+      tmpdir = @tmpdir
+      Dir.chdir(tmpdir) do
+        # tmpdir is ENV["HOME"], so another dir is needed
+        subdir = "#{tmpdir}/mydir"
```

Comment:
For clarify, let's name it `project_dir` instead.

```rb
project_dir = "#{tmpdir}/project"
```

Reply:
Good catch, done 👍